bodyswap.lua

```diff
@@ -1,27 +1,35 @@
 --@ module = true
-
-local utils = require 'utils'
-local validArgs = utils.invert({
-    'unit',
-    'help'
+local dialogs = require('gui.dialogs')
+local utils = require('utils')
+local argparse = require('argparse')
+local makeown = reqscript('makeown')
+
+local args = { ... }
+local options = {
+    help = false,
+    unit = -1,
+}
+
+local positionals = argparse.processArgsGetopt(args, {
```

Comment:
all arg processing should be done after the module check; it shouldn't be run when this script is loaded as a module